src/h/sys.h

```diff
@@ -365,6 +365,11 @@
 #include <GL/glx.h>
 #endif					/* XWindows */
 #include <GL/glu.h>
+#if HAVE_LIBFREETYPE
+   #include <ft2build.h>
+   #include FT_FREETYPE_H
+   #define PNG_SKIP_SETJMP_CHECK /* to fix compile error specified by png.h */
```

Comment:
What is the  png  error ?

Reply:
To be honest, I don't remember. I tried compiling without it, and it runs. I'll remove it.